rw-system.sh

```diff
@@ -309,6 +335,15 @@ if getprop ro.vendor.build.fingerprint | grep -qE '^xiaomi/(daisy|wayne)/(daisy|
     setprop audio.camerasound.force true
 fi
 
+if getprop ro.vendor.build.fingerprint | grep -iq \
+        -e xiaomi/polaris -e xiaomi/sirius -e xiaomi/dipper \
+        -e xiaomi/wayne -e xiaomi/jasmine -e xiaomi/jasmine_sprout \
+        -e xiaomi/platina -e iaomi/perseus -e xiaomi/ysl \
+        -e xiaomi/nitrogen -e xiaomi/daisy -e xiaomi/sakura \
```

Comment:
The list looks quite long, but well we can remove devices if it breaks stuff, so good for me

Reply:
Great to hear. Glad to finally get this approved, it's been bugging a great portion of the Xiaomi user-base for quite some time.

Reply:
Seems not all devices need this.  Goodix fingerprint working well on dipper without this fix.  But wayne/jasmine/whyred/tulip should need this, maybe lavender/violet need this, too.  Because they all made by longcheertel.

Reply:
This file : files/xiaomi-goodixfp.mdt
does not correspond to the name used in scripts : xiaomi-goodixfp.mdp
t -> p